storybookjs/storybook
# [storyshots-puppeteer] Allow specifying an existing browser

## What I did

Allow specifying an existing browser for storyshots-puppeteer - this is useful for integrating storyshots with a headless Chrome instance running on another machine or in a Docker container.

## How to test

Is this testable with Jest or Chromatic screenshots? *There do not appear to be any tests for storyshots-puppeteer at present except for the url builder, so no?*
Does this need a new example in the kitchen sink apps? *Probably not? Please weigh in.*
Does this need an update to the documentation? *Yes, done*

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
